lib/core/src/msParam.cpp

```diff
@@ -1022,6 +1022,28 @@ parseMspForPosInt( msParam_t* inpParam ) {
     return myInt;
 }
 
+int parseMspForStr_new(msParam_t* inpParam, char** outParam)
+{
+    if (inpParam == NULL || inpParam->inOutStruct == NULL) {
```

Comment:
Within the body of the function we can use C++. That means we can also return `nullptr` from the function even in a C context.

Let's use `nullptr` instead of `NULL`.

_This applies to the other locations too._